### DISCUSSION TITLE
About Unit error of Quota dashboard

### DISCUSSION THREAD

**Discussion by qrm1982** [NetApp/harvest#3069] [Q&A]
Hi all,
         Now we find some error about unit error of Quota dashboard,it shows TiB in Space Used column and TB in Space Hard Limit column as attached,how can we set them right to TB?  Thanks.
![unit](https://github.com/user-attachments/assets/91120c98-1a50-42aa-b2b5-c3eb0eed7731)
?

**Comment by Hardikl** [comment]
Yes, unit should be in `TiB` not in `TB`.
This is metric doc for the same.
<img width="817" alt="image" src="https://github.com/user-attachments/assets/71c09cd3-05d5-4b6c-aab2-05cf8a5dd204">

We will update it, By the time you could change the right side corner of Overrides from `kilobytes` to `kibibytes`.
<img width="1787" alt="image" src="https://github.com/user-attachments/assets/58b0d849-a5b8-463d-a049-f838e0c50d0c">

**Comment by Hardikl** [comment]
SM shows this
<img width="1516" alt="image" src="https://github.com/user-attachments/assets/c8d507d4-405d-431f-a388-9690a9017a35">

**Reply by qrm1982** [reply]
Hi Hardikl,
                 Thanks,we have solved the problem.

**Comment by Hardikl** [comment]
We have also made the appropriate changed in PR: https://github.com/NetApp/harvest/pull/3072 for the same.
